ucm2/Intel/avs/avs_max98373/Google-Atlas-1.0-HiFi.conf

```diff
@@ -0,0 +1,30 @@
+SectionDevice."Speaker" {
+	Comment "Speakers"
+
+	Value {
+		PlaybackPCM "hw:${CardId},1"
+		PlaybackMixerElem "DSP"
```

Comment:
It's possible to combine `Left / Right` volume controls to one stereo using alsa-lib's configuration (see `ucm2/codecs/wsa883x/init.conf`) and add this control here. But it may be changed later.